fastparse/shared/src/main/scala/fastparse/ParserHelper.scala

```diff
@@ -0,0 +1,46 @@
+package fastparse
+
+import fastparse.Utils.HexUtils
+
+import scala.collection.mutable.ArrayBuffer
+
+trait ParserHelper[ElemType] {
```

Comment:
We should avoid calling things `Helper`, since it doesn't tell us anything about what it's used for (all code is meant to "help" people...)

Could this be called `ElemTypeMessageFormatter`? Or something else? I'm not entirely sure what would be the best name, but `ParserHelper` is probably too vague